src/components/ArticlesList.jsx

```diff
@@ -0,0 +1,32 @@
+import { getArticles } from "../api";
+import { useState, useEffect } from "react";
+
+const ArticlesList = () => {
+  const [articlesList, setArticlesList] = useState([]);
+
+  useEffect(() => {
+    getArticles().then((articles) => {
```

Comment:
Don't forget to include some way of signposting to a user if the data is loading/if there's been an error with the request. On slow connections this shows a blank screen for a while - not obvious to a user what's going on straight away! If you want to test errors you could just make a get request to an endpoint the doesn't exist in your api